### ISSUE TITLE
Create ARM release for godotenv binary?

### ISSUE THREAD

**Issue by evandam** [joho/godotenv#179]
Hey @joho I was just wondering if it was possible to have an ARM release for this project? It would be great to just install it with curl in environments where golang is not actually installed.

Thanks!

**Comment by joho** [comment]
Yep, I think this is 100% reasonable, will figure out how I can jam it into the current releases building github action.

**Comment by joho** [comment]
Seems someone did it for us in #174 just testing it now

**Comment by rfay** [comment]
Looks like this was fixed in #174 right? But we need a release created?
